client/src/Component/SMSModal.js

```diff
@@ -1,23 +1,63 @@
-/* eslint-disable jsx-a11y/label-has-for */
-import { useState } from "react";
+
+import { useState, useEffect } from "react";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 
+
+let defaultMessage = "Hi, this is BikesforRefugees. Bikes are now available for you to pick up. Please click LINK to choose a pick up date. For any questions, please contact 07900000000.";
```

Comment:
```suggestion
const defaultMessage = "Hi, this is BikesforRefugees. Bikes are now available for you to pick up. Please click LINK to choose a pick up date. For any questions, please contact 07900000000.";
```
Nitpick: We're using `const` by default in the rest of the codebase and we're not reassigning the variable, so we can use `const` here too